src/js/controllers/import.js

```diff
@@ -25,15 +27,15 @@ angular.module('copayApp.controllers').controller('importController',
 
       if ($scope.isChromeApp) {
```

Comment:
Chrome or dev?  I recall this being exactly what was in the baseline I started with but i can add `|| isDevel`.  Is that what you're asking about?